prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -126,17 +132,21 @@ public void onChange(Entity oldEntity, Entity newEntity) throws FalconException{
                 backlogMetricStore.deleteEntityInstance(newProcess.getName());
                 entityBacklogs.remove(newProcess);
             }
+            Process process = EntityUtil.getEntity(oldEntity.getEntityType(), oldEntity.getName());
```

Comment:
As discussed, it is better to knock off a gauge to clearly indicate no metrics are flowing for this process, due to delete.